main.tf

```diff
@@ -12,77 +18,64 @@ module "kms" {
   tags                    = var.tags
 }
 
-# ---------------------------------------------------------------------------------------------------------------------
-# EKS CONTROL PLANE
-# ---------------------------------------------------------------------------------------------------------------------
 module "aws_eks" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "v18.26.6"
+  version = "v18.29.1"
```

Comment:
we don't want to go up to v18.30 if we can avoid it - 18.30 added default tags logic that is proving to be problematic and will be removed/reverted in v19.x